src/liquid/global/styles/theming/index.md

```diff
@@ -0,0 +1,226 @@
+---
+eleventyNavigation:
+  key: Theming
+  parent: Globals
+layout: layout.njk
+title: Theming
+permalink: global/theming/
+---
+
+<link rel="stylesheet" href="/css_components/ld-theme.css">
+<link rel="stylesheet" href="/css_components/ld-button.css">
+
+# Theming
+
+Liquid comes with multiple themes for theming its UI components. Most of the examples in the documentation have a theme switch built in, which you can use to switch between the available themes. In the following examples this switch is turned off. Instead, the examples show how you can apply a theme yourself.
+
+---
+
+## How to apply a theme
+
+You apply a theme by wrapping whatever needs to be themed in an element with a `ld-theme-<themename>` CSS class.
+
+{% example 'html', false, true, false, false %}
+<div class="ld-theme-bubblegum">
+  <ld-button>Text</ld-button>
+</div>
+
+<!-- CSS component -->
+
+<div class="ld-theme-bubblegum">
+  <button class="ld-button">Text</button>
+</div>
+{% endexample %}
+
+## Theme inception
+
+In rare cases you will want to have a theming element wrapped by another theming element. Liquid supports a one level theme inception, which should be sufficiant for most edge cases. You can **not** wrap a theme in a theme in a theme... So, here is an example of a one level theme inception:
```

Comment:
Is this still true at all? At least I can now wrap a button in three themes and the inner one wins over the others. This has to do with CSS custom prop scopes. The inner scope wins over the outer scopes. I think we can simply update the docs by saying that we allow theme inception of any depth.